cra-config.yaml

```diff
@@ -6,8 +6,3 @@ CRA_TARGETS:
     PROFILE_ID: "0e6e7b5a-817d-4344-ab6f-e5d7a9c49520" # SCC profile ID (currently set to the FSCloud 1.4.0 profile).
     CRA_ENVIRONMENT_VARIABLES:
       TF_VAR_disable_public_endpoint: "true"
-      # Add more environment variables as needed
```

Comment:
Made this change here because the CRA scan was failing due to the following error.
```
PROFILE_ID 0e6e7b5a-817d-4344-ab6f-e5d7a9c49520 set from config file cra-config.yaml
Exporting environment variable: TF_VAR_disable_public_endpoint=true
Exporting environment variable: #Addmoreenvironmentvariablesasneeded=null
/__w/_temp/20a955b4-6c77-4028-85d2-b75b794be3d5.sh: line 97: export: `#Addmoreenvironmentvariablesasneeded=null': not a valid identifier
Error: Process completed with exit code 1.
```